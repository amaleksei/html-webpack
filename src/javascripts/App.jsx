// @flow

import React from 'react';

function App() {
  return (
    <div>
      <div className="hello">
        <span>Hello webpack!</span>
      </div>
      <div className="description">
        <span>Boilerplate for a Webpack and React</span>
      </div>
      <div className="image-insert">
        <img alt="" src="./assets/images/test-image.gif" />
      </div>
    </div>
  );
}

export default App;
