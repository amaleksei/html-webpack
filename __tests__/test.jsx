import React from 'react';
import renderer from 'react-test-renderer';
import App from '../src/javascripts/App';

test('App', () => {
  const component = renderer.create(<App />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
