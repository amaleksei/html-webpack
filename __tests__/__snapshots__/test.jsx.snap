// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App 1`] = `
<div>
  <div
    className="hello"
  >
    <span>
      Hello webpack!
    </span>
  </div>
  <div
    className="description"
  >
    <span>
      Boilerplate for a Webpack and React
    </span>
  </div>
  <div
    className="image-insert"
  >
    <img
      alt=""
      src="./assets/images/test-image.gif"
    />
  </div>
</div>
`;
